src/sentry/static/sentry/app/views/performance/transactionSummary/statusBreakdown.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {Location} from 'history';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import {LightWeightOrganization} from 'app/types';
+import BreakdownBars from 'app/components/charts/breakdownBars';
+import {SectionHeading} from 'app/components/charts/styles';
+import Placeholder from 'app/components/placeholder';
+import QuestionTooltip from 'app/components/questionTooltip';
+import EventView from 'app/utils/discover/eventView';
+import DiscoverQuery from 'app/utils/discover/discoverQuery';
+import {getTermHelp} from 'app/views/performance/data';
+import space from 'app/styles/space';
+
+type Props = {
+  organization: LightWeightOrganization;
+  location: Location;
+  eventView: EventView;
+};
+
+function StatusBreakdown({eventView, location, organization}: Props) {
+  const breakdownView = eventView
+    .withColumns([
+      {kind: 'function', function: ['count', '', '']},
+      {kind: 'field', field: 'transaction.status'},
+    ])
+    .withSorts([{kind: 'desc', field: 'count()'}]);
+
+  return (
+    <Container>
+      <SectionHeading>
+        {t('Status Breakdown')}
+        <QuestionTooltip
+          position="top"
+          title={getTermHelp(organization, 'statusBreakdown')}
+          size="sm"
+        />
+      </SectionHeading>
+      <DiscoverQuery
+        eventView={breakdownView}
+        location={location}
+        orgSlug={organization.slug}
+      >
+        {({isLoading, tableData}) => {
+          if (isLoading) {
+            return <Placeholder height="150px" />;
+          }
+          if (!tableData) {
+            return 'oh no';
+          }
+          const points = tableData.data.map(row => ({
```

Comment:
Is it possible that `data` is an empty array? If it is, would this just be blank?